src/compute/src/render/reduce.rs

```diff
@@ -628,11 +628,22 @@ where
     {
         let error_logger = self.error_logger();
 
+        let output_name = format!(
+            "ReduceInaccumulable Distinct{}",
+            if std::any::TypeId::of::<R>() == std::any::TypeId::of::<()>() {
```

Comment:
This is complex for me to parse. Please use a `name` parameter instead that supplies a suffix to this operator's name---it'd avoid the surprise @def- encountered!

Reply:
Sounds good, done!